packages/search/addon/services/navi-search/navi-report-and-dashboard-search-provider.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * This service is used to search for reports and dashboards stored in the persistence layer.
+ */
+
+import { inject as service } from '@ember/service';
+import NaviBaseSearchProviderService from '../navi-base-search-provider';
+
+export default class NaviReportAndDashboardSearchProviderService extends NaviBaseSearchProviderService {
```

Comment:
Would it be better to separate the search providers for reports and dashboards? Whats the benefit from using just one?